src/SumoCoders/FrameworkCoreBundle/Resources/views/base.html.twig

```diff
@@ -44,9 +44,6 @@
       <script type="text/javascript" src="{{ asset('/bundles/mopabootstrap/js/mopabootstrap-collection.js') }}"></script>
       <script type="text/javascript" src="{{ asset('/assets/js/sumo_plugins.js') }}"></script>
 
-      {% if bundleExists('SumoCodersFrameworkSearchBundle') %}
-        <script type="text/javascript" src="{{ asset('/assets/js/frameworksearchbundle.framework.form.search.js') }}"></script>
```

Comment:
https://github.com/sumocoders/FrameworkSearchBundle/blob/convert-coffeescript-to-es6/Resources/assets/js/FrameworkSearch.js#L75